Lib/tkinter/test/test_tkinter/test_widgets.py

```diff
@@ -522,7 +522,11 @@ def test_selection_methods(self):
         self.assertEqual(widget.selection_get(), '2345')
         widget.selection_adjust(0)
         self.assertEqual(widget.selection_get(), '12345')
-        widget.selection_adjust(0)
+        self.assertEqual(widget.selection_element(), "none")
```

Comment:
Since this method differs enough from other selection methods, I suggest to add a separate test method for it.